corelib/crayfish_trace.h

```diff
@@ -0,0 +1,53 @@
+#ifndef CRAYFISH_TRACE_H
+#define CRAYFISH_TRACE_H
+
+#include <QSize>
+#include <QVector>
+#include <QPointF>
+#include <QLineF>
+
+class Output;
+class MapToPixel;
+struct RendererConfig;
+
+typedef QVector<QPointF> TraceStreamLine;
+
+class TraceRendererCache {
```

Comment:
I thought the cache would contain interpolated vectors for each pixel, so that we can animate any particle we want, so the algorithm could randomly add/remove particles that are being animated, rather than having to stick with fixed stream lines...

(rate of adding of particles, particle's alive period and fade-out rate would be configurable)